data/json/items/melee.json

```diff
@@ -4602,6 +4602,29 @@
    },
    {
     "type" : "AMMO",
+    "id" : "material_rocksalt",
+    "category" : "spare_parts",
+    "price" : 50,
+    "name" : "rock salt",
+    "name_plural" : "rock salt",
+    "symbol" : "=",
+    "color" : "gray",
+    "description" : "A handful of rock salt crystals.  Could be refined into table salt.",
+    "material" : "stone",
+    "volume" : 1,
+    "weight" : 14,
+    "bashing" : 1,
+    "cutting" : 0,
+    "to_hit" : 0,
+    "ammo_type" : "components",
+    "damage" : 0,
+    "range" : 0,
+    "dispersion" : 0,
+    "recoil" : 0,
+    "count" : 50
```

Comment:
At 50 charges per 1 volume, and 4 charges per 8 charges of salt, a single unit of volume of rock salt "contains" as much table salt as 12.5 units of volume of table salt.